src/actions/setIsMulticursorExecuting.ts

```diff
@@ -0,0 +1,30 @@
+import ActionType from '../@types/ActionType'
+import CommandId from '../@types/CommandId'
+import State from '../@types/State'
+import Thunk from '../@types/Thunk'
+
+interface SetIsMulticursorExecutingPayload {
+  value: boolean
+  commandType?: ActionType | CommandId
```

Comment:
`ActionType` and `CommandType` are very different things. I don't think we should ever union that together.

Please try to find a different approach that does not combine these types.